docs/vhost-user-net-testing.md

```diff
@@ -0,0 +1,130 @@
+The purpose of this document is to illustrate how to test vhost-user-net in cloud-hypervisor with OVS/DPDK as backend.
+
+### Framework
+
+It's just a basic function test to validate the communication between two virtual machines by connecting to vhost-user port respectively provided by OVS/DPDK.
+```
+             +----+----------+          +-------------+-----------+-------------+          +----------+----+           
+             |    |          |          |             |           |             |          |          |    |           
+             |    |vhost-user|----------| vhost-user  |   ovs     | vhost-user  |----------|vhost-user|    |           
+             |    |net device|          | port 1      |           | port 2      |          |net device|    |           
+             |    |          |          |             |           |             |          |          |    |           
+             |    +----------+          +-------------+-----------+-------------+          +----------+    |           
+             |               |          |                                       |          |               |           
+             |vm1            |          |                  dpdk                 |          |           vm2 |           
+             |               |          |                                       |          |               |           
+          +--+---------------------------------------------------------------------------------------------+--+        
+          |  |                                       hugepage                                              |  |        
+          |  +---------------------------------------------------------------------------------------------+  |        
+          |                                                                                                   |        
+          |                                              host                                                 |        
+          |                                                                                                   |        
+          +---------------------------------------------------------------------------------------------------+        
+```
+### Prerequisites
+
+Prior to run the test, the following steps need to be done, [vhost-user-ovs-dpdk](https://wiki.qemu.org/Documentation/vhost-user-ovs-dpdk) is a good reference which tells how to do.
+- Enable hugepage
+- Install kernel-devel
+- Build DPDK from source
+- Build OVS from source
+
+### Test
+The test will run with mq support enabled, 2 queue pairs defined for both OVS and virtual machine. Here is the details how we can run the test.
+
+_Setup OVS_
+
+ovs_test.sh is created to setup and start OVS. OVS will provide the dpdkvhostuser type port which is as server mode.
+```
+#!/bin/bash
```

Comment:
I would remove this line.

Reply:
Remove "#!/bin/bash", is that what you mean?